src/components/CreateClusterTemplateWizard/Steps/ManageQuotasStep/QuotaCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Form } from '@patternfly/react-core';
+// import { PlusIcon } from '@patternfly/react-icons';
+import { useFormikContext } from 'formik';
+import React from 'react';
+import SelectField, { SelectInputOption } from '../../../../helpers/SelectField';
+import { useTranslation } from '../../../../hooks/useTranslation';
+import NewQuotaDialog from '../../../ClusterTemplateQuotas/NewQuotaDialog/NewQuotaDialog';
+import { useQuotas } from '../../../../hooks/useQuotas';
+import { QuotaOptionObject, WizardFormikValues } from '../../types';
+import BudgetField from '../../../ClusterTemplateQuotas/NewQuotaDialog/BudgetField';
+import { PlusIcon } from '@patternfly/react-icons';
+import get from 'lodash/get';
+import { useAddAlertOnError } from '../../../../alerts/useAddAlertOnError';
+
+type QuotaCardProps = {
+  quotaIdx: number;
+  fieldName: string;
+};
+
+const getQuotaOptionObject = (name: string, namespace: string): QuotaOptionObject => ({
+  name: name,
+  namespace: namespace,
+  toString: () => name,
+});
+
+const QuotaCard = ({ quotaIdx, fieldName }: QuotaCardProps) => {
+  const { values, setFieldValue, errors } = useFormikContext<WizardFormikValues>();
+  const quotaFieldName = `${fieldName}.quota`;
+
+  const checkboxFieldName = `${fieldName}.limitAllowed`;
+
+  const { t } = useTranslation();
+  const [quotasContext, loaded, error] = useQuotas();
+  useAddAlertOnError(error, t('Failed to load quota options'));
+  const [newQuotaDialogOpen, setNewQuotaDialogOpen] = React.useState(false);
+  const getSelectOptions = (): SelectInputOption[] => {
+    return quotasContext.getAllQuotasDetails().map((quotaDetails) => ({
+      value: getQuotaOptionObject(quotaDetails.name, quotaDetails.namespace),
+      disabled: false,
+      description: t('Namespace: {{namespace}}', { namespace: quotaDetails.namespace }),
+    }));
+  };
+
+  const selectOptions = React.useMemo(() => getSelectOptions(), [quotasContext]);
```

Comment:
we should setup eslint as it does not complain that `getSelectOptions` is missing from deps